schemas/main/accesscredential.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
Is this folder going to be regenerated with each PR?

Reply:
It isn't at present, I've flipped flopped on if this should be against the main branch, latest tag or a random commit tbh.

The benefit of following main is that we can immediately consume new things added in main, the downside is that we'd need to somehow limit this to a specific tag/commit when we generate a release from common-instancetypes to ensure CI continues to work.